src/data/text/move_descriptions.h

```diff
@@ -2618,9 +2618,61 @@ static const u8 sPLASMA_FISTSDescription[] = _(
 	"Normal moves become Electric.");
 
 static const u8 sPHOTON_GEYSERDescription[] = _(
-	"The foe's highest offensive\n"
+	"The user's highest offensive\n"
```

Comment:
Feel like either this or the next line is missing "stat"; maybe "The user's highest stat determines its category" if space is a concern